### ISSUE TITLE
support https

### ISSUE THREAD

**Issue by nerilex** [PrishtinaHackerspace/HACCSY#2]
Communication with the CRM-Server should use an _encrypted channel_ (eg. https) in my opinion.

Especially if the device is connected via WiFi and the access credentials to the network are known, it is not too hard to get illegitimate access to the facility.

**Comment by altinukshini** [comment]
Yup, I totally agree! Will have to do it... -  I left this for when we move to a stable version of the CRM, and have our website use https as well...
Btw we switched to cable connection a while back, because wifi was not reliable and it was slow.
